apps/docker-compose.yml

```diff
@@ -21,3 +20,56 @@ services:
       ports:
         - $AGILE_CLIENT_PORT:$AGILE_CLIENT_PORT
       restart: always
+
+    agile-osjs:
+      image: agileiot/agile-osjs-armv7l
+      ports:
+        - 8000:8000/tcp
+      restart: always
+
+    agile-nodered:
+      image: agileiot/agile-nodered-armv7l
+      hostname: agile-nodered
+      volumes:
+        - $HOME/.agile/nodered:/root/.node-red
+      ports:
+        - 1880:1880/tcp
+      links:
+        - agile-core:agile
+        - agile-influxdb:db
+      restart: always
+
+    agile-influxdb:
```

Comment:
This is the data server right that I am building right or are you using influx for something else?